### PR TITLE
rhn_register: add sslcacert and systemorgid options

### PR DESCRIPTION
**Pull request by mmatuska**
In addition to serverUrl we are actively using the sslCACert and systemorgid options

### CODE REVIEW

**Comment by gregdek** [comment]
@barnabycourt @vritant @flossware Please review this PR to make sure it adheres to the following guidelines:

http://docs.ansible.com/developing_modules.html#module-checklist

If it passes these guidelines, and if you believe it’s a good PR otherwise, please add a comment with "shipit" in the text, and we will flag it for inclusion.

If it fails these guidelines, please add a comment with "needs_revision" in the text, along with the needed changes, and we will flag it for revision.

**Comment by gregdek** [comment]
@barnabycourt @vritant @flossware This change is still pending your review; do you have time to take a look and comment? Please comment with text 'shipit' or 'needs_revision' as appropriate.

File: packaging/os/rhn_register.py [hunk 1]
```diff
@@ -62,6 +62,18 @@
         required: False
         default: null
         version_added: "2.0"
+    sslcacert:
+        description:
+            - supply a custom ssl CA certificate file for use with registration
+        required: False
+        default: null
```
Comment (barnabycourt) [inline packaging/os/rhn_register.py]: please use None instead of null

File: packaging/os/rhn_register.py [hunk 2]
```diff
@@ -62,6 +62,18 @@
...
+        version_added: "2.0"
+    systemorgid:
+        description:
+            - supply an organizational id for use with registration
+        required: False
+        default: null
```
Comment (barnabycourt) [inline packaging/os/rhn_register.py]: please use None instead of null

**Comment by barnabycourt** [comment]
needs_revision
@mmatuska Just two simple updates for the newer guidelines on return values and this will be good to go. Thanks for your submission!

**Comment by mmatuska** [comment]
I have updated the pull request as suggested. To follow your suggested standard I recommend you change all the other "null" documentation fields to "None" as well.

**Comment by barnabycourt** [comment]
shipit

**Comment by robynbergeron** [comment]
Thanks again to @mmatuska for this PR, and thanks @barnabycourt @vritant @flossware for reviewing. Marking for inclusion.

**Pull request merged**